Sources/Rendering/Core/Camera/index.js

```diff
@@ -348,7 +348,26 @@ function vtkCamera(publicAPI, model) {
   publicAPI.getFrustumPlanes = (aspect) => {
     // Return array of 24 params (4 params for each of 6 plane equations)
   };
-  publicAPI.getCameraLightTransformMatrix = () => {};
+
+  publicAPI.getCameraLightTransformMatrix = () => {
+    const matrix = mat4.create();
+    mat4.copy(matrix, model.cameraLightTransform);
+    return matrix;
+  };
```

Comment:
I'm not seeing anything call this yet, so maybe we can change the signature. I'm trying to avoid heap allocations. So we could have it take matrix as an argument and then it just does the copy.

Reply:
I forgot to modify updateLightsGeometryToFollowCamera in Core/Renderer, which calls this function. Just added it to commit.